docs/pages/technology/saas.rst

```diff
@@ -0,0 +1,47 @@
+.. _saas_considerations:
+
+SaaS Considerations
+###################
+
+While the Nuts node itself is small and lightweight, there are operational aspects of running a Nuts node
+which could make parties consider to outsource hosting:
+
+- Backing up data (both on-disk and in databases) and testing backup restore procedures
+- Keeping the node up-to-date (patching)
+- Securely handling private key material to avoid theft or data loss
+- Key rotation
+
+Additional Services
+^^^^^^^^^^^^^^^^^^^
+
+Additional services could involve;
+
+- a UI for DID and credential administration
+- middleware to support implementation of Nuts use cases (Zorgtoepassingen, e.g. eOverdracht),
+  involving more in-depth integration with e.g. customer's FHIR servers.
+
+Multi-tenancy
+^^^^^^^^^^^^^
+
+The Nuts node itself is not multi-tenant, meaning someone with access to its API can read and use any credential present on it.
+This is a problem if the hoster of the node (e.g. SaaS vendor) does not control the applications using the API (care organization or EHR vendor).
+
+By running a Nuts node for each customer, Therefore, it's recommended to run a separate Nuts node for each customer.
```

Comment:
This sentence needs some rewriting.

Reply:
I would also not recommend it at this point in time. It would be better to facilitate using multi-tenant middleware.

Reply:
Make sure you define *customer*. The first time I read it, I thought care organization.